client/my-sites/people/people-list-item/index.jsx

```diff
@@ -60,11 +64,14 @@ export default React.createClass( {
 				this.props.onRemove &&
 				<div className="people-list-item__actions">
 					<button className="button is-link people-list-item__remove-button" onClick={ this.props.onRemove }>
-						{ this.translate( 'Remove', { context: 'Verb: Remove a user or follower from the blog.' } ) }
+						{ this.props.translate( 'Remove', { context: 'Verb: Remove a user or follower from the blog.' } ) }
```

Comment:
@akirk I don't think I have a great understanding of how much belongs in `context` vs. `comment`. Would it be good here to add both and leave `context` as `verb`